src/main/java/com/lob/api/Configuration.java

```diff
@@ -52,7 +52,24 @@ public static ApiClient getConfigForIntegration() {
 
         HttpBasicAuth basicAuth = (HttpBasicAuth) apiClient.getAuthentication("basicAuth");
         String LOB_API_KEY = getEnvVariable("LOB_API_TEST_KEY");
-        
+
+        apiClient.setBasePath(BASE_PATH);
+        basicAuth.setUsername(LOB_API_KEY);
+        return apiClient;
```

Comment:
@jho44  What is the purpose of all the extra code here? It seems like both use these lines.

Reply:
One of the functions uses the TEST KEY for integration tests while the other (previously undefined) uses the LIVE KEY for integration tests. I was following precedent with older SDK tests. But in hindsight, it would've been DRYer to pass "live" or "test" as a parameter to one function that chooses between the keys based on the parameter. Should I do that instead? I'll need to edit all other files that use the `getConfigForIntegration` function.